README.md

```diff
@@ -57,148 +52,77 @@ Please follow the [installation procedure](#installation--usage) and then run th
+              new \GuzzleHttp\Client([
+                  'timeout'  => 30,
+                  'connect_timeout' => 30,
+                  'read_timeout' => 30
+              ]), $configuration);
 
-// Setting applicant details
-
-$applicantDetails = new Onfido\Model\ApplicantRequest();
+```
 
-$applicantDetails->setFirstName('Jane');
-$applicantDetails->setLastName('Doe');
-$applicantDetails->setDob('1990-01-31');
+### Making a call to the API
 
-$address = new \Onfido\Model\Address();
-$address->setBuildingNumber('100');
-$address->setStreet('Main Street');
-$address->setTown('London');
-$address->setPostcode('SW4 6EH');
-$address->setCountry('GBR');
+```php
+try {
+  $applicant = $onfidoApi->createApplicant(
+      new Onfido\Model\ApplicantBuilder(
+          [
+              'first_name' => 'First',
+              'last_name' => 'Last'
+          ])
+  );
+
+  // To access the information use the getter for the desired property on the object, for example:
+  $applicant->getFirstName();
+
+  // ...
+
+} catch (Onfido\ApiException $ex) {
+  // Handle API exception
+  echo 'Caught exception: ',  $ex->getMessage(), "\n";
+}
+```
 
-$applicantDetails->setAddress($address);
+### Webhook event verification
 
-// Setting check details
+Webhook events payload needs to be verified before it can be accessed. Library allows to easily decode the payload and verify its signature before returning it as an object for user convenience:
 
-$checkData = new Onfido\Model\CheckRequest();
-$checkData->setReportNames(array('identity_standard'));
+```php
+try {
+  $verifier = new Onfido\WebhookEventVerifier('_ABC123abc...3ABC123_');
 
-// Create an applicant and then a check with an Identity report
+  $signature = 'a0...760e';
 
-try {
-    $applicantResult = $apiInstance->createApplicant($applicantDetails);
-    $applicantId = $applicantResult->getId();
-    print_r("Applicant ID: ".$applicantId);
-    $checkData->setApplicantId($applicantId);
-    $checkResult = $apiInstance->createCheck($checkData);
-    print_r($checkResult);
-} catch (Exception $e) {
-    print_r($e->getResponseBody());
+  $event = $verifier->readPayload('{"payload":{"r...3"}}', $signature);
+} catch (Onfido\OnfidoInvalidSignatureError $ex) {
+  // Invalid webhook signature
 }
+```
 
-?>
+## Contributing
 
-```
+This library is automatically generated using [OpenAPI Generator](https://openapi-generator.tech) - version: 7.4.0;
+therefore all the contributions, except tests files, should target [Onfido OpenAPI specification repository](https://github.com/onfido/onfido-openapi-spec/tree/master) instead of this repository.
 
-## Documentation for API Endpoints
-
-All URIs are relative to *https://api.eu.onfido.com/v3.6*
-
-Class | Method | HTTP request | Description
------------- | ------------- | ------------- | -------------
-*DefaultApi* | [**cancelReport**](docs/Api/DefaultApi.md#cancelreport) | **POST** /reports/{report_id}/cancel | This endpoint is for cancelling individual paused reports.
```

Comment:
Wouldn't it be worth it to keep this documentation?

Reply:
In every language I mean

Reply:
I don't think so, added value is really small.